bundles/WebToPrintBundle/src/Event/DocumentEvents.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\WebToPrintBundle\Event;
+
+final class DocumentEvents
+{
+    /**
+     * Processor contains the processor object used to generate the PDF
+     *
+     * Arguments:
+     *  - processor | instance of the PDF processor Pimcore\Web2Print\Processor\{ProcessorName}
+     *
+     * @Event("Pimcore\Event\Model\DocumentEvent")
+     *
+     * @var string
+     */
+    const PRINT_PRE_PDF_GENERATION = 'pimcore.document.print.prePdfGeneration';
```

Comment:
not sure, shall we change the "namespace" too?
something like
```suggestion
    const PRINT_PRE_PDF_GENERATION = 'pimcore.bundle.webtoprint.document.print.prePdfGeneration';
```

Reply:
I am not sure either, but it would be a breaking change then right?

Reply:
we didn't do that for the other bundles, so it would be fine, at least is consistent